### PR TITLE
Add Derive prisma schema

### PR DESCRIPTION
**Pull request by jefag**

### CODE REVIEW

File: src/components/DelegateStatement/DelegateStatementForm.tsx [hunk 1]
```diff
@@ -106,6 +112,7 @@ export default function DelegateStatementForm({
       delegateStatement: values,
       signature,
       message: serializedBody,
+      scwAddress,
```
Comment (mcgingras) [inline src/components/DelegateStatement/DelegateStatementForm.tsx:115]: does this mean every new tenant gets scwAddress even if its never used?

File: src/lib/viem.ts [hunk 2]
```diff
@@ -25,6 +58,17 @@ export const getWalletClient = (chainId: number) => {
         chain: cyber,
         transport: custom(window.ethereum!),
       });
+    case lyra.id:
```
Comment (mcgingras) [inline src/lib/viem.ts:61]: should we chance lyra to derive?

File: src/lib/viem.ts [hunk 3]
```diff
@@ -1,8 +1,41 @@
...
+  createWalletClient,
+  custom,
+  defineChain,
+  http,
+} from "viem";
+import { cyber, lyra, mainnet, optimism, scroll, sepolia } from "viem/chains";
+
 import "viem/window";
 
+export const lyraTestnet = /*#__PURE__*/ defineChain({
```
Comment (mcgingras) [inline src/lib/viem.ts:12]: same here -- maybe change all instances of lyra just so it's not confusing

File: src/lib/tenant/tenantTokenFactory.ts [hunk 4]
```diff
@@ -38,6 +38,13 @@ export default class TenantTokenFactory {
           decimals: 18,
         };
 
+      case TENANT_NAMESPACES.DERIVE:
+        return {
+          name: "CHANGEME - Derive",
```
Comment (mcgingras) [inline src/lib/tenant/tenantTokenFactory.ts:43]: CHANGEME?

File: src/lib/tenant/configs/ui/ens.ts [hunk 5]
```diff
@@ -22,6 +22,13 @@ export const ensTenantUIConfig = new TenantUI({
     title: "ENS Foundation",
   },
 
+  smartAccountConfig: {
```
Comment (mcgingras) [inline src/lib/tenant/configs/ui/ens.ts:25]: why is ENS getting this?

File: src/lib/tenant/configs/ui/derive.ts [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+        imageTitle: "Voter on Agora",
+        imageDescription:
+          "Delegate your voting power to a trusted representative",
+      },
+    },
+    {
+      route: "proposals",
+      title: "Agora is the home of Dervice delegates",
+      description:
+        "Dervie voters are the stewards for the DAO. You can see them all below, delegate your votes to them, or contact them about your ideas.",
```
Comment (mcgingras) [inline src/lib/tenant/configs/ui/derive.ts:88]: spelling Dervie -> Derive

File: src/lib/tenant/configs/ui/derive.ts [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+    type: "LightAccount",
+    salt: BigInt(0),
+  },
+
+  pages: [
+    {
+      route: "/",
+      title: "Agora is the home of Derive governance",
+      description:
+        "Derive governance is launching now. Start by claiming your token and joining Discourse to engage in the discussion. Delegation and voting are coming soon.",
```
Comment (mcgingras) [inline src/lib/tenant/configs/ui/derive.ts:48]: How does someone claim token? It's okay for us to go live with "claim token"?

File: src/lib/tenant/configs/contracts/derive.ts [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+} from "@/lib/contracts/generated";
+import { ITokenContract } from "@/lib/contracts/common/interfaces/ITokenContract";
+import { TenantContract } from "@/lib/tenant/tenantContract";
+import { TenantContracts } from "@/lib/types";
+import { IGovernorContract } from "@/lib/contracts/common/interfaces/IGovernorContract";
+import { JsonRpcProvider } from "ethers";
+import { lyra } from "viem/chains";
+import { defineChain } from "viem";
+
+export const lyraTestnet = /*#__PURE__*/ defineChain({
```
Comment (mcgingras) [inline src/lib/tenant/configs/contracts/derive.ts:13]: more lyra

File: src/components/Dialogs/DelegateDialog/DelegateButton.tsx [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+}
+// NOT USED IN PRODUCTION TENANT
+export const DelegateButton = ({ delegate, delegator, onChange }: Props) => {
+  const { contracts } = Tenant.current();
+
+  const { data, writeContract, isError: isCancelled } = useWriteContract();
+  const { isLoading, isSuccess, isError } = useWaitForTransactionReceipt({});
+
+  console.log(isLoading, isSuccess, isError);
+  console.log(isCancelled);
```
Comment (mcgingras) [inline src/components/Dialogs/DelegateDialog/DelegateButton.tsx:19]: logs

**Review by mcgingras — APPROVED** [review]

File: src/components/Dialogs/DelegateDialog/DelegateButton.tsx (hunk 9 above)
Comment (andreitr) [reply src/components/Dialogs/DelegateDialog/DelegateButton.tsx:19]: That button isn't used in the app.

**Pull request merged**